library/src/main/scala/scala/scalajs/js/Any.scala

```diff
@@ -126,6 +126,24 @@ object Any extends LowPrioAnyImplicits {
 }
 
 trait LowPrioAnyImplicits {
+  @inline implicit def fromJBoolean(value: java.lang.Boolean): Any =
+    value.asInstanceOf[Any]
+  @inline implicit def fromJByte(value: java.lang.Byte): Any =
+    value.asInstanceOf[Any]
+  @inline implicit def fromJShort(value: java.lang.Short): Any =
+    value.asInstanceOf[Any]
+  @inline implicit def fromJInteger(value: java.lang.Integer): Any =
+    value.asInstanceOf[Any]
+
+  @inline implicit def fromJLong(value: java.lang.Long): Any =
+    if (value eq null) null
+    else value.doubleValue.asInstanceOf[Any]
+
+  @inline implicit def fromJFloat(value: java.lang.Float): Any =
+    value.asInstanceOf[Any]
+  @inline implicit def fromJDouble(value: java.lang.Double): Any =
+    value.asInstanceOf[Any]
+
```

Comment:
No, we don't have the conversion `Char` to `js.Any`, so we shouldn't have `java.lang.Character` to `js.Any` either.